src/commands/info/source.ts

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config();
+require('discord.js');
+import { CommandInteraction } from 'discord.js';
+
+module.exports = {
+  /* @param {Client} client
+   * @param {Interaction} interaction */
+  callback: (interaction: CommandInteraction) => {
+    console.log('Get Source');
+    interaction.reply(process.env.SOURCE);
```

Comment:
I get the following error when I attempt to run the bot.
```sh
There was an error: TSError: ⨯ Unable to compile TypeScript:
src/commands/info/source.ts:11:23 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(options: InteractionReplyOptions & { fetchReply: true; }): Promise<Message<boolean>>', gave the following error.
    Argument of type 'string | undefined' is not assignable to parameter of type 'InteractionReplyOptions & { fetchReply: true; }'.
      Type 'undefined' is not assignable to type 'InteractionReplyOptions & { fetchReply: true; }'.
        Type 'undefined' is not assignable to type 'InteractionReplyOptions'.
  Overload 2 of 2, '(options: string | InteractionReplyOptions | MessagePayload): Promise<InteractionResponse<boolean>>', gave the following error.
    Argument of type 'string | undefined' is not assignable to parameter of type 'string | InteractionReplyOptions | MessagePayload'.
      Type 'undefined' is not assignable to type 'string | InteractionReplyOptions | MessagePayload'.

11     interaction.reply(process.env.SOURCE);
                         ~~~~~~~~~~~~~~~~~~
```

I have this value set in my `.env` file as:

```env
SOURCE = 'https://github.com/stinkyfi/MaddenBot'
```

Do you not experience the same issue?

Reply:
There is no error in `opent-ticket.ts`. I found that copying what that file is doing and concat an empty string solved the errror. 
```ts 
interaction.reply(process.env.SOURCE + '');
```
however during runtime of the `/source` command in discord, I get this error.

```sh
There was an error running this command: TypeError: interaction.reply is not a function
```

I will continue to debug later, just wanted to report this bug to see if you can replicate

Reply:
I will get on my pc to check rn. Also good morning, I guess.

Reply:
Okay step by step : 
- I did not have `SOURCE` as an example in .env and being new and all I kinda never added it in the first place.
- Second, the overloads are simply because types are `string | undefined` you need to add a `!` to the parts that have a possible undefined OR define their type `as string`.
- Thirdly yes I had a similar issue where the things inside .env were not being read hence why you are getting this error.
```
There was an error running this command: TypeError: interaction.reply is not a function
``` 
But yes I will try to replicate right now I hope I am actually useful after a coffee 🤣